### ISSUE TITLE
Screen 18:9

### ISSUE THREAD

**Issue by userkilled** [agateau/pixelwheels#28]
I found such a thing that the start screen is 16:9, but the gameplay itself starts at 18:9.  I have an 18:9 screen.  Xiaomi Redmi 6A bought.

**Comment by OffifialMITX** [comment]
On my 16:9 display it's still cropped. I think it's more like 2:1.

**Comment by agateau** [comment]
Indeed the game tries to use as much screen space as possible. To me it's more a feature than a bug. Is it an issue for you?

**Comment by OffifialMITX** [comment]
![](https://i.imgur.com/4u7hGeT.jpg)
We're talking about the main menu, or, specifically the black bars at the sides.

**Comment by agateau** [comment]
Right, that looks like a minor problem... I am more concerned by the look of the title on your screenshot. Something went really wrong there :scream:. Is this the F-Droid build?

**Comment by OffifialMITX** [comment]
Yes it is. What's wrong with it?
-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by agateau** [comment]
It should look like this:
![Title](https://agateau.com/projects/pixelwheels/0.6.0/title.png)

Anyway, that's another problem. Going to open a separate issue for it.
